tests/test_raffle.py

```diff
@@ -13,11 +13,12 @@
 )
 
 soli_user: models_core.CoreUser | None = None
```

Comment:
Please remove the soli user and the group soli from the group_types, as it is no longer used in the module.